Source/scrollrt.cpp

```diff
@@ -188,17 +188,13 @@ static void scrollrt_draw_cursor_item(const CelOutputBuffer &out)
 
 	const auto &sprite = GetInvItemSprite(pcurs);
 	const int frame = GetInvItemFrame(pcurs);
+	void (*drawingFun)(const CelOutputBuffer &, int, int, const CelSprite &, int) = &CelClippedDrawSafeTo;

```

Comment:
This is extremely ugly to me.... having to define the entire signature parameters like that...

Is there no other way, perhaps using `auto`?

Reply:
no